source/usage-examples/find.txt

```diff
@@ -3,3 +3,54 @@ Find Multiple Documents
 =======================
 
 .. default-domain:: mongodb
+
+You can find multiple documents in a collection by using the ``Find()``
+method.
+
+The following example passes a query filter to the ``Find()`` method,
+which matches documents in the ``zips`` collection if the value of their
```

Comment:
```suggestion
which matches documents in the ``zips`` collection where the value of the
```

Suggestion: I think "matches documents if..." sounds slightly off, as it sounds like the database only goes through the act of matching if a certain condition is met (it always goes through the act of matching, rather some things do not fit the criteria it is looking for).

Suggestion: "their ``pop`` field" sounded strange to me, as it made me think on first reading that the group of documents returned shared a ``pop`` field.

Reply:
accepted